src/leveldbwrapper.h

```diff
@@ -168,6 +199,44 @@ class CLevelDBWrapper
     {
         return pdb->NewIterator(iteroptions);
     }
+
+    /**
```

Comment:
I implemented these in the header to maintain consistency with much of the rest of `CLevelDBWrapper`, which is also defined here. I'll move what's below over to the `.cpp` file, but we should probably think about moving the rest of it out of the header at some point.